spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/FlywayEndpoint.java

```diff
@@ -38,21 +42,32 @@
  * @since 1.3.0
  */
 @ConfigurationProperties(prefix = "endpoints.flyway")
-public class FlywayEndpoint extends AbstractEndpoint<List<FlywayMigration>> {
+public class FlywayEndpoint extends AbstractEndpoint<Map<String, List<FlywayMigration>>> {
 
-	private final Flyway flyway;
+	private final List<Flyway> flyway;
 
-	public FlywayEndpoint(Flyway flyway) {
+	public FlywayEndpoint(List<Flyway> flyway) {
```

Comment:
You're breaking backward compat there. Can you please add the constructor with a single resource back and just call the other one with `Collections.singletonList` ?